wiki/Ranking_Criteria/Beatmap_Nominator_Rules/en.md

```diff
@@ -0,0 +1,56 @@
++ **You are free to pop a rebubble of another nominator if you are not the original bubble popper and disagree with the reasoning for disregarding the bubble pop.** Thus the original popper is unable to overvote two other nominators.
++ **If the community member(s) requesting disqualification cannot reach an agreement with the mapper and you placed the last bubble or heart on a map, you cannot rebubble or requalify it.** For osu!taiko, osu!catch, osu!mania, and hybrid sets, you may break these conditions once per map due to the low number of nominators.
++ **Do not violate the terms of services or the osu! rules and the Code of Conduct for Modding and Mapping.** Misbehaving in public can and will get you expelled from the Beatmap Nomination Group.
++ **Do not nuke beatmaps in any case.** While you are able to set this icon, nuking a beatmap indicates that it is “so wrong” that it will never reach a rankable state and should hence not be used by Nominators. If you accidentally set the icon, contact a QAT immediately or overwrite the icon in case you wanted to post a bubble or heart.
+
+## Changelog
+This section lists recent addition to this ruleset as well as rules that are in the process of transition and may be handled differently for a certain time. This interval as well as additional information, where available, is listed in _italics_.
+
++ **Do not qualify osu! beatmaps of any kind unless you are a nominator in the osu! advanced tier.** This includes osu! and another mode hybrid sets. When nominating hybrid sets including modes you are not too familiar with as an advanced tier nominator, collaborate with a nominator from the respective gamemode(s).
+
+_Addition date: 06th April 2017, 6 months of transition time (previously bubbled maps are exempt), **affects all sets on 6th October 2017**_
++ **Do not nominate beatmaps of game modes you are not formally qualified for.** You must be qualified to nominate at least one of the game modes included in the mapset by being a member of the respective part of the Beatmap Nomination Group in order to be allowed to nominate it.
+
+_Addition date: 06th April 2017, **in effect immediately**_
++ **__Every__ mapset is approved through two nominations** Other factors such as the mapset itself being a Marathon, a normal spread or its length do not matter, the amount of nominations required in order to move it to the Qualified section remains at two. More nominations than necessary must not be done.
+
+_Addition date: 02nd June 2017, **in effect immediately** (maps at two bubbles by beginner nominators have to find an advanced tier BN and are allowed to break this rule by going for a third nomination to transition, for all others an advanced tier nominator or an equivalent of that can qualify immediately)_
```

Comment:
wrongly placed `_` (should be after immediately and not after the whole paragraph)